lib/python/rose/resource.py

```diff
@@ -140,9 +140,9 @@ def get_util_name(self, separator=" "):
         util = self.util
         try:
             if namespace is None:
-                namespace = os.getenv("ROSE_NS")
+                namespace = os.environ("ROSE_NS")
             if util is None:
-                util = os.getenv("ROSE_UTIL")
+                util = os.environ("ROSE_UTIL")
```

Comment:
`os.getenv` is a function but `os.environ` is a dict, so need to update `(...)` to `[...]`.

Reply:
Derp!